### PR TITLE
use correct context for build and push action

### PR DESCRIPTION
**Pull request by pkubatrh**
I expect this should fix the failures you are seeing @zmiklank

### CODE REVIEW

File: .github/workflows/build-and-push.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ jobs:
           registry_username: ${{ secrets[matrix.quayio_username] }}
           registry_token: ${{ secrets[matrix.quayio_token] }}
           dockerfile: ${{ matrix.dockerfile }}
-          docker_context: ${{ matrix.dockerfile_path }}
```
Comment (zmiklank) [inline .github/workflows/build-and-push.yml:61]: Isn't it the same as `docker_context: '.'`?

Comment (pkubatrh) [reply .github/workflows/build-and-push.yml:61]: Good point, I would expect that to be the same. Updated.

**Review by zmiklank — APPROVED** [review]
thanks. let's see.

**Comment by zmiklank** [comment]
It seems, that it works. Now the problem is, that the repositories for micro images do not exist yet in quay.io.

**Pull request merged**